core/package.json

```diff
@@ -101,6 +102,7 @@
     "url": "ssh://github.com/kyma-project/luigi.git"
   },
   "dependencies": {
+    "documentation": "^11.0.0",
```

Comment:
Since documentation was added, there should also be changes in the `package-lock.json` file, but seems the file did not get checked in.

Reply:
Done.